legal-api/src/legal_api/services/filings/validations/common_validations.py

```diff
@@ -197,6 +197,30 @@ def validate_pdf(file_key: str, file_key_path: str) -> Optional[list]:
     return None
 
 
+def validate_pdf_no_page_size(file_key: str, file_key_path: str) -> Optional[list]:
```

Comment:
No need of new function. Add a new parameter into existing one to `validate_pdf(file_key: str, file_key_path: str, verify_paper_size:bool=True)`

```
if verify_paper_size:
    # Check that all pages in the pdf are letter size and able to be processed.
    if any(x.mediaBox.getWidth() != 612 or x.mediaBox.getHeight() != 792 for x in pdf_reader.pages):
        msg.append({'error': _('Document must be set to fit onto 8.5” x 11” letter-size paper.'),
                    'path': file_key_path})
```